src/Sylius/Component/Promotion/Factory/ActionFactory.php

```diff
@@ -63,4 +63,16 @@ public function createPercentageDiscount($percentage)
 
         return $action;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function createPercentageShippingDiscount($percentage)
+    {
+        $action = $this->createNew();
+        $action->setType('shipping_discount');
```

Comment:
Please add a comment about this in the mentioned issue, so we remember to change that :)